api/api_dataset_rest_proxy.py

```diff
@@ -79,7 +77,9 @@ async def post(self, request: Request):
                 status_code=403,
             )
         async with httpx.AsyncClient() as client:
-            respon = await client.post(url, json=payload_json, headers=dict(request.headers))
+            headers = headers = dict(request.headers)
```

Comment:
can you remove the extra headers =
```suggestion
             headers = dict(request.headers)
```

Reply:
upsi, sure!